gui/shredder/cmdline.py

```diff
@@ -107,7 +107,7 @@ def parse_arguments(root_logger):
     )
 
     try:
-        parser.parse_args()
+        parser.parse_args([ arg for arg in sys.argv if not arg=="--gui" ])
```

Comment:
I just thought it was neater to do this here rather than https://github.com/sahib/rmlint/blob/master/lib/cmdline.c#L214-L215.  It also makes the `rmlint --gui` command order-independent i.e. `rmlint --show-settings --gui` works (ok, that's kinda pointless).